active/0000-anonymous-amd.md

```diff
@@ -0,0 +1,45 @@
+- Start Date: (fill me in with today's date, 2016-06-14)
+- RFC PR: (leave this empty)
+- ember-cli Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes extending the `app.import` API to consume anonymous AMD modules. It accompanies [ember-cli PR 5976](https://github.com/ember-cli/ember-cli/pull/5976).
+
+# Motivation
+
+AMD modules come in two flavors: named and anonymous. _Anonymous AMD_ is the more portable distribution format, but it typically requires preprocessing into _named AMD_ before it can be included into an application. Today, ember-cli users can add arbitrary third-party _named AMD_ modules into their application via:
+
+    app.import('/path/to/module.js');
+
+But this does not support _anonymous AMD_ modules, which is annoying because _anonymous AMD_ is the better format for library distribution and is widely used.
+
+# Detailed design
+
+In order to de-anonymize AMD, it's necessary to choose a name for the module for use within a given application. So I propose extending the:
+
+    app.import('/path/to/module.js');
+
+API with an additional argument:
+
+    app.import('/path/to/module.js', { amdModule: 'some-dep' });
```

Comment:
Remapping a bunch of _named_ AMD modules to other names seems like a pretty different (and niche) concern. And multiple _anonymous_ AMD modules in one bundle seems like invalid input.

Of the variants, `{ amd: 'some-dep' }` seems fine, `{ amd: { id: 'some-dep' }}` can be supported cleanly in the future if we ever need to expand the arguments, and `{is: 'amd', as: 'some-dep' }` invites two undefined cases (when either property appears alone).